packages/client/hmi-client/src/views/DocumentView.vue

```diff
@@ -97,6 +134,29 @@ if (previousOpenTabs) {
 	<section v-else class="recent-documents-page">
 		<resources-list :project="props.project" :resource-route="RouteName.ModelRoute" />
 	</section>
+
+	<slider-panel
+		class="slider"
+		content-width="300px"
+		tab-width="56px"
+		direction="right"
+		header="Notes"
+		v-model:is-open="isPanelOpen"
+	>
+		<template v-slot:content>
+			<div v-for="(annotation, idx) of annotations" :key="idx" class="annotation-panel">
+				<p>{{ annotation.content }}</p>
+				<div class="annotation-footer">
+					<div>{{ annotation.username }}</div>
+					<div>{{ formatDate(annotation.timestampMillis) }}</div>
+				</div>
+			</div>
+			<div class="annotation-panel">
+				<Textarea v-model="annotationContent" rows="5" cols="30" aria-labelledby="annotation" />
+				<Button @click="addAnnotation()" label="Add note" />
+			</div>
```

Comment:
Why not put this in the footer slot?

Reply:
The footer is for metadata ... kind of like a bibliography kind of thing.